pkg/spdk/replica.go

```diff
@@ -251,6 +281,11 @@ func (r *Replica) construct(bdevLvolMap map[string]*spdktypes.BdevInfo) (err err
 	switch r.State {
 	case types.InstanceStateStopped, types.InstanceStatePending:
```

Comment:
Ah, could you update it to `case types.InstanceStatePending:`? 
I added the stopped case in https://github.com/longhorn/longhorn-spdk-engine/pull/284.

Reply:
I fixed this issue as well in commit https://github.com/longhorn/longhorn-spdk-engine/pull/278/commits/a748606b225d9fc94690bed203ffb34f5801b071

Would you take a look again?